app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -24,31 +24,23 @@ def description
   end
 
   def send_report!
```

Comment:
The `send_abuse_report!` method (and the `ArchiveConfig.ABUSE_REPORTS_SITE` setting?) was just added as a temporary measure when Abuse and Support were using different trackers. Since they're back to sharing the same bug site, I think we can get rid of `send_abuse_report!` and use this for both support and abuse reports to keep our code DRYer.